src/game/time_attack/results.c

```diff
@@ -94,7 +94,7 @@ u32 CreateTimeAttackResults(u32 finishTime)
 
     for (i = 0; i < 7; i++) {
         s = &resultsCutScene->unk178[i];
-        s->x = i * 14 + 72;
+        s->x = i * 14 + ((DISPLAY_WIDTH / 2) - 46);
```

Comment:
I think this should be `((DISPLAY_WIDTH / 2) - 48)`, not `-46`.

Reply:
My math is stupid.